fmf/utils.py

```diff
@@ -214,15 +214,23 @@ def filter(filter, data, sensitive=True, regexp=False):
     """
     Return true if provided filter matches given dictionary of values
 
-    Filter supports disjunctive normal form with '|' used for OR, '&'
-    for AND and '-' for negation. Individual values are prefixed with
-    'value:', leading/trailing white-space is stripped. For example::
+    Filter supports disjunctive normal (DNF) form with '|' used for OR,
```

Comment:
```suggestion
    Filter supports disjunctive normal form (DNF) with '|' used for OR,
```

Let's put the abbreviation after the full term?